content/en/tracing/troubleshooting/_index.md

```diff
@@ -2,444 +2,109 @@
 title: APM Troubleshooting
 kind: documentation
 further_reading:
+- link: "/tracing/troubleshooting/tracer_startup_logs/"
+  tag: "Documentation"
+  text: "Datadog tracer startup logs"
+- link: "/tracing/troubleshooting/tracer_debug_logs/"
+  tag: "Documentation"
+  text: "Datadog tracer debug logs"
 - link: "/tracing/troubleshooting/agent_apm_metrics/"
   tag: "Documentation"
   text: "APM metrics sent by the Datadog Agent"
 ---
 
-When experiencing unexpected behavior with Datadog APM, there are a few common issues you can look for before reaching out to [Datadog support][1]:
-
-1. **Make sure the Agent has APM enabled**:
```

Comment:
I think it still makes sense to keep some mention of the agent in this doc since it is required under most circumstances.  If we're removing this in favor of pointing to other documentation we should likely reference that doc here in my opinion.